hudi-client/src/main/java/org/apache/hudi/config/HoodieMetricsConfig.java

```diff
@@ -58,6 +59,12 @@
 
   public static final String GRAPHITE_METRIC_PREFIX = GRAPHITE_PREFIX + ".metric.prefix";
 
+  // User defined
+  public static final String USER_DEFINED_REPORTER_PREFIX = METRIC_PREFIX + ".user.defined";
+  public static final String USER_DEFINED_REPORTER_CLASS = USER_DEFINED_REPORTER_PREFIX + ".class";
+
+  public static final String DEFAULT_USER_DEFINED_REPORTER_CLASS = DefaultUserDefinedMetricsReporter.class.getName();
```

Comment:
DEFUALT would be empty string. also please refer to https://github.com/apache/hudi/blob/5e7ab11e2ead23428ed5089421f2abd6433fe8e5/hudi-client/src/main/java/org/apache/hudi/config/HoodieIndexConfig.java#L42

Reply:
Good idea, got it